components/collections/import-export-collections.vue

```diff
@@ -204,6 +205,10 @@ export default {
             name: "",
             requests: [],
           }
+      if (folders) {
+        //pick up collection name even when all children are folders
+        postwomanCollection[0].name = collection.info ? collection.info.name : ""
```

Comment:
FYI: This is a bugfix for the case where a collection that contains only folders is imported, and in that case the name for taht collection  will not be populated.